tests/utils_tests/test_autoreload.py

```diff
@@ -19,6 +19,16 @@
 
 class TestFilenameGenerator(SimpleTestCase):
 
+    @classmethod
```

Comment:
I don't like that because now tests aren't independent as shown by the tests that have to clean up after themselves with `os.remove()`, correct? I think we can just leave this file alone for now, especially considering the autoload stuff may be refactored at some point ([#27685](https://code.djangoproject.com/ticket/27685)).